### PR TITLE
add drug terms best practices to contributing doc

### PR DESCRIPTION
**Pull request by allaway**
Closes issue #235

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -16,6 +16,42 @@ If you are proposing a new term, then we require a source for the definition. Th
...
 2. If you are not (nor are you working with a Sage Bionetworks supported community) it is up to you for a strategy for controlling new terms to be added.
 
+## Guidelines for specific term types 
+
+In some situations (e.g. drug names), terms are not always well-captured by the ontologies found in the Ontology Lookup Service. We've defined some best practices for contributing these terms here.
+
+### Contribution of drug terms
+
+The preferred first-pass strategy for chemical name annotation is to search the EMBL-EBI ontology lookup service to find names, descriptions, and sources. Typically, the NCI Thesaurus will provide a suitable description for drugs and other biologically active molecules. In situations where the query molecule is not found in EMBL-EBI OLS, MeSH (https://meshb.nlm.nih.gov/) is a helpful secondary location to find chemical descriptions.
+
```
Comment (sgosline) [inline CONTRIBUTING.md]: Can you add link to OLS here just for completion?

**Comment by allaway** [comment]
@sgosline made the requested change!

**Comment by sgosline** [comment]
Looks good, go ahead and merge/close!

**Comment by allaway** [comment]
closes #235

**Pull request merged**